usr/share/rear/layout/save/default/305_uniq_disktodo.sh

```diff
@@ -0,0 +1,5 @@
+# Script 305_uniq_disktodo.sh
```

Comment:
Do we need to name the script *in* the script? Makes renaming/moving harder

Reply:
Good point - removed 1st line